mango/processing/__init__.py

```diff
@@ -5,12 +5,16 @@
     is_json_file,
```

Comment:
We are missing a statement:
`from .date_functions_2 import *`

Reply:
IS there any reason to have the date functions split between two files instead of just one file?

Reply:
no, they could go in the same file.